packages/main/src/mainWindow.ts

```diff
@@ -95,7 +95,9 @@ async function createWindow(): Promise<BrowserWindow> {
     }
 
     if (import.meta.env.DEV) {
-      browserWindow?.webContents.openDevTools();
+      if (!process.env.CLOSE_DEVTOOLS || process.env.CLOSE_DEVTOOLS !== 'true') {
```

Comment:
That does not sound logic: I would rather have OPEN_DEVTOOLS=false to prevent opening DevTools. And test should be merged with previous test